PeerioTechnologies/peerio-desktop
# feat(notifications): focus on input when clicking notification

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### PR author

#### Relevant issue tracker links  
https://peerio.tpondemand.com/entity/7720

#### Linked PRs in other repos
... 
#### Manual testing instructions  

0. Put focus outside of message input (e.g. click on messages)
1. Click on notification.
2. Check that message input is focused.

#### Additional notes
... 

----
### Reviewer

* Localisation.
* Architectural decisions.
* Performance.
* Comments.
* Big feature tech design documentation.
* Readability.
* DRY.
* Can we extract something to SDK?
* Are there any bicycles invented that should be replaced with existing SDK
* Are there unit and e2e tests added?

----
### Repository owner
